### PR TITLE
Feature/table validation example

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes PXBLUE-2322 .

#### Changes proposed in this Pull Request:
- Form validation Table view (Desktop)
- Form Validation List View (Mobile)

#### To Test:
- Local Test URL: http://localhost:4200/forms-and-validation/in-a-table

### CODE REVIEW

File: src/app/pages/forms-and-validation/in-a-list/in-a-list.module.ts [hunk 1]
```diff
@@ -22,6 +23,7 @@ import { InAListComponent } from './in-a-list.component';
         BrowserAnimationsModule,
```
[the remark this answers is not in the document]
Comment (ektasawant) [reply src/app/pages/forms-and-validation/in-a-list/in-a-list.module.ts:23]: after removing `BrowserAnimationsModule` getting following error:

image

[the remark this answers is not in the document]
Comment (emclaug2) [reply src/app/pages/forms-and-validation/in-a-list/in-a-list.module.ts:23]: Ah okay my mistake, I didn't run the tests.

**Comment by ektasawant** [comment]
> I do think the switch between table and list is happening prematurely. I'd match react here and switch at the `xs` (600px) viewport.

Done Please review.

**Comment by emclaug2** [comment]
Thanks for updating the breakpoint changes, this looks really good.  There's just one more small thing that I noticed whenever I refactored the react example this morning.  The margin above the divider of the last "Max" InfoListItem should be increased a little bit.
image


I handled this on the react side by introducing a 4th InfoListItem that has an empty `title` prop so it renders empty, but has the `partial` divider.  Then you can easily set a marginTop/marginBottom to get the alignment to match the specs.

**Comment by ektasawant** [comment]
> Thanks for updating the breakpoint changes, this looks really good. There's just one more small thing that I noticed whenever I refactored the react example this morning. The margin above the divider of the last "Max" InfoListItem should be increased a little bit.
> image
> 
> I handled this on the react side by introducing a 4th InfoListItem that has an empty `title` prop so it renders empty, but has the `partial` divider. Then you can easily set a marginTop/marginBottom to get the alignment to match the specs.

I have fixed following, please review

1. Alignment of App bar title with InfoListItem title
2. Add empty InfoListItem for matching specs.

**Review by emclaug2 — APPROVED** [review]

**Comment by emclaug2** [comment]
Looking good!

**Pull request merged**